vendor.mod

```diff
@@ -119,6 +119,8 @@ require (
 	tags.cncf.io/container-device-interface v1.0.1
 )
 
```

Comment:
Looks like go mod did a whoopsie and moved this out of the group (it often needs a bit of manual intervention)

Reply:
Whoops; meant to comment on the line below